bootstrap.sh

```diff
@@ -64,3 +64,23 @@ do
     && cp -n Local.templates/$CONFIG_FILE Local/$CONFIG_FILE \
   )
 done
+
+# Build Yubikit
+YUBIKIT_DIR=Carthage/Checkouts/yubikit-ios
+YUBIKIT_OUT=ThirdParty/YubiKit
+
+rm -rf YUBIKIT_DIR
+rm -rf YUBIKIT_OUT
+
+if [ ! -d "$YUBIKIT_DIR" ]; then
+  git clone https://github.com/jumde/yubikit-ios/ $YUBIKIT_DIR
+fi
+
+SRCDIR=$PWD
+mkdir -p $SRCDIR/$YUBIKIT_OUT/include
+cd $YUBIKIT_DIR/
```

Comment:
You can utilize `pushd` and `popd` here instead of `cd`ing into YUBIKIT_DIR and cd'ing back to src:

```
pushd $YUBIKIT_DIR
# pwd is now $YUBIKIT_DIR
popd
# pwd is now whatever it was before pushd was called
```